static/src/javascripts/projects/common/modules/commercial/acquisitions-link-enrichment.js

```diff
@@ -60,10 +61,14 @@ const addReferrerDataToAcquisitionLinksOnPage = (): void => {
         fastdom.read(() => el.getAttribute('href')).then(link => {
             if (link) {
                 fastdom.write(() => {
-                    el.setAttribute(
-                        'href',
-                        addReferrerDataToAcquisitionLink(link)
+                    let linkWithAcquisitionData = addReferrerDataToAcquisitionLink(
+                        link
                     );
+                    if (link.contains('support.theguardian.com/contribute'))
```

Comment:
I think you get away with it because the following line is a single statement (despite the link breaks) - but I'd definitely prefer some curly braces here!

Reply:
👍